### DISCUSSION TITLE
How to replace the circular marker with a "double circular" marker

### DISCUSSION THREAD

**Discussion by gchiozzi** [Stellarium/stellarium#3363] [General]
Sorry, I am new to programming around Stellarium and I am trying to understand the APIs and scripting.

Stellarium provides a circular marker around the center of the field of view with a configurable diameter in degrees.

I would need instead a marker with an inner and outer circles (inside the inner circle is the "science field of view" while the ring between the outer and inner circle is the area patroled by guide probes).

What would be the correct way of implementing this?
Can I do it just with scripts? Are there any good examples?
Or i would have to modify the source code and change the implementation of the circular marker or duplicate it?

Thanks for the help.

**Comment by alex-w** [comment]
I'm probably wrong, but please check the Oculars plugin

**Reply by gchiozzi** [reply]
THANKS!
What you say is correct: the Telrad in Oculars does exactly what I need (even with four rings).
From the Oculars buttons I can do it.
I will now look at how to control the size and to toggle the Telrad on/off from the Remote Control plugin: I just need to discover the correct commands and parameters to send to stellarium.

Thanks again!

**Reply by gchiozzi** [reply]
Ok. I am almost there :-)
With the python script that I attach (actually coming from a jupyter notebook) I can toggle the Telrad and retrieve both its status and the diameter of the four FOV circles.

What is strange is that **the last post command to SET the FOV does not work**: I do not get any error, but nothing changes.
I am probably doing something stupid :-)
Any idea?

See the attached code :-(
[ControlStellariumTelrad.py.txt](https://github.com/Stellarium/stellarium/files/12323075/ControlStellariumTelrad.py.txt)
